keps/sig-storage/177-volume-snapshot-validation/kep.yaml

```diff
@@ -0,0 +1,54 @@
+title: CSI Snapshot
```

Comment:
I would call the directory "177-volume-snapshot" since it contains the overall snapshot feature